README.md

```diff
@@ -116,6 +116,27 @@ though of course it doesn’t produce a binary you can run.
 
 ### Building for Android target
 
+Prerequisites:
+Servo's build system assumes that both the Android SDK & NDK are
```

Comment:
It'd be great to have a small set of Android-specific instructions here like we have for other platforms. I'm happy to write this.

Reply:
I am happy to write this.  Also, there is a link at the top of this README to servo wiki page with more [detailed instructions](https://github.com/servo/servo/wiki/Building-for-Android), but this is out of date. Do we want to add Android-specific instructions here or keep them in the wiki?